hawkular-wildfly-agent/src/main/java/org/hawkular/agent/monitor/extension/MonitorServiceRestartParentAttributeHandler.java

```diff
@@ -58,15 +65,39 @@ protected boolean requiresRuntime(OperationContext context) {
     @Override
     protected void removeServices(OperationContext context, ServiceName parentService, ModelNode parentModel)
```

Comment:
Let me know if you would like to have this on Util (or somewhere else) to execute blocks of code only once in a `OperationContext`.